packages/adapter/src/request-building/parameters.ts

```diff
@@ -76,18 +76,29 @@ function buildUserParameters(options: CachedBuildRequestOptions): BuilderParamet
   }, initalParameters());
 }
 
+function buildRelayMetadataParameters(options: CachedBuildRequestOptions): BuilderParameters {
+  const { parameters } = options;
+
+  const parameterKeys = Object.keys(parameters).filter((key) => key.startsWith('_airnode_'));
```

Comment:
I don't think this is ideal...

What about rather adding a new option to the `BuildRequestOptions`, specifically for parameters that bypass the OIS checks and always get included?

Reply:
I thought about that as well but it seemed like too big of a change for some reason. I thought of adding something like this:

```
readonly metadata: {
        readonly [key: string]: string;
    }
```
    
to the BuildRequestOptions interface instead of using the same `parameters` object. Is that what you're suggesting or did I get it wrong?

Reply:
Yeah that's what I was thinking. It's not really ideal either but it kinda feel better than just filtering on the parameter name including `_airnode_`. That feels like it might have unintended consequences at some point

Reply:
Done. I like this much better since I agree that searching by startsWith('_adapter_') might lead to unexpected issues.

I'm still not sure if double checking the metadata parameters is required in the adapter package. Meaning looking for the relay metadata reserved parameter value and checking the metataParameters props based on the specified version. It seems unnecessary but I'd like your opinion.